samples/int/nc.bas

```diff
@@ -0,0 +1,121 @@
+' -- #FUJINET NETCAT Example --
+
+' Default unit # for connection
+CONN=2
+MODE=12
+TRANS=0
+
+' URL for connection
+URL$=""
+
+' RX Buffer
+DIM BUF(8192) BYTE
+
+' Procedures '''''''''''''''''''''''''
+
+PROC BANNER
+
+PRINT "** NETCAT IN FASTBASIC **"$9B
+ENDPROC
+
+PROC GETCONN
+  DO 
+
+    INPUT "URL: ",URL$
+
+    C$=URL$[1,2]
+
+    IF C$="N:"
+      EXIT
+    ENDIF 
+
+  LOOP
+
+  INPUT "TRANS (0=none, 1=CR, 2=LF, 3=CR/LF): ",TRANS
+  
+ENDPROC
+
+PROC INTCLR
+  POKE $D302, PEEK($D302) & 127
+ENDPROC
+
+PROC CONNECT
+
+  PRINT "Connecting to:"
+  PRINT URL$
+
+  NOPEN CONN,MODE,TRANS,URL$
+  @INTCLR
+  NSTATUS CONN
+  
+ENDPROC
+
+PROC IN
+  NSTATUS CONN
+  BW = DPEEK($02EA)
+
+  IF BW = 0
+    EXIT
+  ENDIF
+
+  IF BW > 8192
+    BW = 8192
+  ENDIF 
+
+  NGET CONN,&BUF,BW
+  
+  FOR I=0 TO BW-1
+    PRINT CHR$(BUF(I));
+  NEXT I
+  
+ENDPROC
+
+PROC OUT
+  GET K
+  NPUT CONN,&K,1
+  POKE $2FC,255
+ENDPROC
+
+PROC NC
+
```

Comment:
I don't know if an `@INTCLR` here, would be usefull just before entering the loop. What happes if you call `@IN` when FujiNet does not expect it? If it is harmless, it would be better to call it more than necessary, so without the `@INTCLR` would be better.